client/app/hosting/cdn/order/hosting-cdn-order.controller.js

```diff
@@ -30,7 +30,7 @@ angular
           }
           $scope.loading.model = true;
           HostingOptionOrder.getOrderEnums('hosting.web.CdnOfferEnum').then((models) => {
-            if (!$scope.isPerfOffer()) {
+            if (!$scope.isPerfOffer() and !$scope.isCloudWebOffer() ) {
```

Comment:
you're calling this function without offer argument ?

Reply:
You're right, one definition is missing in the local controller of order cdn part.

I just duplicate the behavior of isPerfOffer

Reply:
(and fixed)

Reply:
I also see my two branches was mixed up. For this part, please refer to my another PR (1128)